include/flatbuffers/flatbuffers.h

```diff
@@ -463,6 +494,10 @@ class FlatBufferBuilder FLATBUFFERS_FINAL_CLASS {
   // Get the serialized buffer (after you call Finish()).
   uint8_t *GetBufferPointer() const { return buf_.data(); }
 
+  // Get the released pointer to the serialized buffer.
+  // Don't attempt to use this FlatBufferBuilder afterwards!
+  unique_ptr_t Release() { return buf_.release(); }
```

Comment:
`Release()` or `ReleaseBufferPointer()`? The latter is consistent with the `Get` version.